dashboard/app/helpers/locale_helper.rb

```diff
@@ -34,8 +34,6 @@ def options_for_locale_select
     options
   end
 
-  private
```

Comment:
See: http://stackoverflow.com/questions/12835367/why-can-private-helper-methods-still-be-accessed-in-views

> Helpers are modules that get mixed in to the views. This means that public, protected and private methods in the helper become public, protected and private methods on the views.
>
> I don't think that you can actually hide the helper methods from the view. You'd need to do something like have a helper class which you instantiate in the helper and then delegate calls to that - sounds like it could get messy fast though. :)